var/spack/repos/builtin/packages/arborx/package.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Arborx(CMakePackage):
+    """ArborX is a performance-portable library for geometric search"""
+
+    homepage = "http://github.com/arborx/arborx"
+    git      = "https://github.com/arborx/arborx.git"
+
+    version('master', branch='master')
```

Comment:
This version should be called `develop`.
But also... we cannot merge until you've added at least one CHECKSUMMABLE version to the package.  Since ArborX hasn't made any releases, this means you'll have to choose a git hash and give it a version number.  Since you are a maintainer of `ArborX`, my recommendation is you create a release at `AroborX`, and then use it here.

Reply:
This version should be called `master`, since that's what the `arborx` developers refer to it as. Now that #1983 has been merged, Spack treats `master` and `develop` similarly.

Reply:
@tgamblin have we developed any standards on whether or not a checksummable release is required? There are already quite a few packages in Spack that don't have one. If we do develop a standard that requires this, someone should add checksummable releases to those other packages.